platform/mellanox/mlnx-platform-api/sonic_platform/fan.py

```diff
@@ -46,9 +46,11 @@
 FAN_DIR = "/var/run/hw-management/thermal/fan{}_dir"
 FAN_DIR_VALUE_EXHAUST = 0
 FAN_DIR_VALUE_INTAKE = 1
+FAN_DIR_VALUE_NOT_APPLICABLE = 2
```

Comment:
which one will be better, 'N/A' or 'UNKNOWN'?

Reply:
The schema values support 'N/A', and also the 'FanBase' interface refers to this value as not applicable. So I think the 'N/A' value is better.